rclcpp/src/rclcpp/context.cpp

```diff
@@ -23,25 +23,46 @@
 
 #include "rcl/init.h"
 #include "rcl/logging.h"
+
 #include "rclcpp/detail/utilities.hpp"
 #include "rclcpp/exceptions.hpp"
 #include "rclcpp/logging.hpp"
+
+#include "rcutils/error_handling.h"
+#include "rcutils/macros.h"
+
 #include "rmw/impl/cpp/demangle.hpp"
 
+#include "./logging_mutex.hpp"
+
 /// Mutex to protect initialized contexts.
 static std::mutex g_contexts_mutex;
 /// Weak list of context to be shutdown by the signal handler.
 static std::vector<std::weak_ptr<rclcpp::Context>> g_contexts;
 
 using rclcpp::Context;
 
-static
-std::shared_ptr<std::mutex>
-get_global_logging_configure_mutex()
+extern "C"
 {
-  static auto mutex = std::make_shared<std::mutex>();
-  return mutex;
+void
+rclcpp_logging_output_handler(
```

Comment:
this is intended to be called from somewhere else? if that so, interface should be exposed. if not, make it static?

Reply:
Done in https://github.com/ros2/rclcpp/pull/1125/commits/bfc98d0e0be5fc89b3bc15be01885e4434f4fc48.